### PR TITLE
CHG pdf export working [MACRO-2277]

### PR DESCRIPTION
**Pull request by synoet**
Seems like pdf exporting works without pdfium + without making any major changes.
Only made a small fix to the worker saveAs function + added some testing ui + logic in qa-env to test pdf saving

### CODE REVIEW

File: qa-env/src/OfficeDocument/OfficeDocument.tsx
Comment (chase) [inline qa-env/src/OfficeDocument/OfficeDocument.tsx:1]: OfficeDocument is the component used as the general document viewer, it is not the app integrating the view of the document.

This belongs in https://github.com/coparse-inc/lok-wasm/blob/main/qa-env/src/App.tsx

OfficeDocument should always be in a state where we can just copy it into the monorepo without modifications.

File: qa-env/src/OfficeDocument/utils.ts
Comment (chase) [inline qa-env/src/OfficeDocument/utils.ts:1]: Similar to the comment above, this belongs in the top level, this is not used to display and interact with the document.

File: libreoffice-core/desktop/wasm/worker.ts [hunk 1]
```diff
@@ -64,8 +64,9 @@ const handler: AsyncMessage = {
   ): Promise<ArrayBuffer> {
     const { readUnlink } = await lokPromise;
     const tmpFile = `/${Date.now()}.${format}`;
-    byRef(ref)?.saveAs(`file://${tmpFile}`, format);
-    return readUnlink(tmpFile);
+    byRef(ref)?.saveAs(`file://${tmpFile}`, format, undefined);
```
Comment (chase) [inline libreoffice-core/desktop/wasm/worker.ts:71]: What's the deal with the `undefined` at the end? A comment as to _why_ this change was necessary would be a great addition.

It should work correctly without it, but maybe embind doesn't handle optional the way it's documented: https://github.com/coparse-inc/lok-wasm/blob/main/libreoffice-core/desktop/source/app/main_wasm.cxx#L57-L58

**Review by chase — CHANGES_REQUESTED** [review]

File: libreoffice-core/desktop/wasm/worker.ts (hunk 1 above)
Comment (synoet) [reply libreoffice-core/desktop/wasm/worker.ts:71]: Added a comment

**Review by chase — APPROVED** [review]

**Pull request merged**